dnsimple/templates.go

```diff
@@ -128,6 +128,22 @@ func (s *TemplatesService) DeleteTemplate(accountID string, templateID string) (
 	return templateResponse, nil
 }
 
+// ApplyTemplate deletes a template.
+//
+// See https://developer.dnsimple.com/v2/templates/domains/#apply
+func (s *TemplatesService) ApplyTemplate(accountID string, domainID string, templateID string) (*TemplateResponse, error) {
+	path := versioned(fmt.Sprintf("%v/templates/%v", domainPath(accountID, domainID), templateID))
```

Comment:
You should reuse the [`templatePath`](https://github.com/dnsimple/dnsimple-go/blob/master/dnsimple/templates.go#L26) here.